.github/workflows/build.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and push container image
+
+on:
+  push:
+    branches:
+      - main
+
```

Comment:
```suggestion
  pull_request:
```

Reply:
Similar to the changes made on forest-offset-fires, I think we can consolidate this to one workflow. Rather than uncommenting NO_PUSH to enable building and pushing on PRs, we can comment/uncomment this line.